app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,30 @@
+<nav class="navbar is-dark" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="/">
+      <i class="fas fa-dollar-sign is-size-5"></i>
+      <h1 class="has-text-weight-semibold has-text-white-ter">Budget Tracker</h1>
+    </a>
+
+    <a role="button" id="navbar-burger" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
```

Comment:
I think this camel case ID came from Bulma, but it would be more conventioned to dash case it.

Reply:
If I remove this ID or change it to camelcase, the navbar no longer works on mobile view, and since there are no other hits for this term in my code, it seems to be baked in to Bulma.

Reply:
Good to know! As you were LG x